### PR TITLE
Handle errors

### PR DESCRIPTION
**Pull request by ieva-ts**
* Upgraded dependencies
* Removed android-only 'loadAccount'
* Implemented returning detailed errors
* Fix 'signOut' to return result to Flutter on all completions

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -65,8 +62,9 @@ class _MyAppState extends State<MyApp> {
   Future<void> _signOut() async {
     String authToken;
     try {
-      authToken = await this.fma.signOut;
-    } on PlatformException catch(e) {
+      await this.fma.signOut;
+      authToken = "";
```
Comment (PetrasBartusis) [inline example/lib/main.dart]: single qoute string

File: example/lib/main.dart [hunk 2]
```diff
@@ -96,39 +85,38 @@ class _MyAppState extends State<MyApp> {
...
+                children: <Widget>[
+                  RaisedButton(
+                    onPressed: _acquireTokenInteractively,
+                    child: Text('Acquire Token'),
+                  ),
+                  RaisedButton(onPressed: _acquireTokenSilently, child: Text('Acquire Token Silently')),
+                  RaisedButton(onPressed: _signOut, child: Text('Sign Out')),
+                  RaisedButton(onPressed: _fetchMicrosoftProfile, child: Text('Fetch Profile')),
+                  SizedBox(
+                    height: 8,
```
Comment (PetrasBartusis) [inline example/lib/main.dart]: I'd reccomend to remove comma and format

File: example/lib/main.dart [hunk 3]
```diff
@@ -96,39 +85,38 @@ class _MyAppState extends State<MyApp> {
...
+                  ),
+                  SizedBox(
+                    height: 8,
```
Comment (PetrasBartusis) [inline example/lib/main.dart]: I'd reccomend to remove comma and format

File: example/lib/main.dart [hunk 4]
```diff
@@ -96,39 +85,38 @@ class _MyAppState extends State<MyApp> {
...
+                  ),
+                  Text("Profile: $_msProfile"),
+                  SizedBox(
+                    height: 8,
```
Comment (PetrasBartusis) [inline example/lib/main.dart]: I'd reccomend to remove comma and format

**Review by PetrasBartusis — APPROVED** [review]

File: example/lib/main.dart (hunk 1 above)
Comment (ieva-ts) [reply example/lib/main.dart]: Didn't cleanup example project, has no analysis too; just fixed to accommodate lib changes and formatted.

Updated to single quote.

File: example/lib/main.dart (hunk 2 above)
Comment (ieva-ts) [reply example/lib/main.dart]: Formatted

File: example/lib/main.dart (hunk 3 above)
Comment (ieva-ts) [reply example/lib/main.dart]: Formatted

File: example/lib/main.dart (hunk 4 above)
Comment (ieva-ts) [reply example/lib/main.dart]: Formatted

**Pull request merged**